example/src/main/scala/example/package.scala

```diff
@@ -14,10 +14,8 @@
  * limitations under the License.
  */
 
-import zio.Has
-
 package object example {
-  type ContributorsCache = Has[ContributorsCache.Service]
+  type ContributorsCache = ContributorsCache.Service
```

Comment:
I think we should get rid of old module pattern encoding as well.

Reply:
I will use case class